Source/DataSources/CzmlDataSource.js

```diff
@@ -375,6 +377,9 @@ define([
         case VerticalOrigin:
             return VerticalOrigin[defaultValue(czmlInterval.verticalOrigin, czmlInterval)];
         default:
+            if (type && type.defaultValue)  {
```

Comment:
I'm not sure why we need this.  I would rather see you add a case statement for `Angle` instead of this new `defaultValue`property.